nextcord/ui/button.py

```diff
@@ -134,7 +134,7 @@ def style(self) -> ButtonStyle:
         return self._underlying.style
 
     @style.setter
-    def style(self, value: ButtonStyle):
+    def style(self, value: ButtonStyle) -> None:
```

Comment:
I don't think it's necessary to annotate return types of functions that do not have return statements. It can be inferred that the return type is None.

Reply:
```
nextcord.ui.button.Button.label
  /.../nextcord/ui/button.py:error: Return type annotation is missing
```
pyright thought otherwise (`pyright --verifytypes nextcord`)

Reply:
Are there ways to tell it not to check those rules? It seems weird to annotate None when the function doesn't return anything.

Reply:
Similar reasoning to other comment, but there is a tool to do this automatically  [autotyping](https://pypi.org/projects/autotyping)